main/.vuepress/config.js

```diff
@@ -45,7 +45,13 @@ module.exports = {
     ]
   ],
 
-  plugins: ['check-md'],
+  plugins: [
+    'check-md',
+    [
+      '@vuepress/last-updated',
+      { dateOptions: { dateStyle: "medium", timeStyle: "long", timeZone: "Etc/UTC" } },
```

Comment:
This updates the page footers from ambiguous patterns like "_**Last Updated**: 2/20/2022, 1:50:55 AM_" to "_**Last Updated**: Feb 20, 2022, 1:50:55 AM UTC_".